acceptance/tests/commands/args-handling.rb

```diff
@@ -1,17 +1,7 @@
 test_name "argument handling" do
   bin_loc = puppetdb_bin_dir(database)
 
-  ["ssl-setup", "import", "export"].each do |k|
-    step "running puppetdb #{k} -h should not error out" do
-      on database, "#{bin_loc}/puppetdb #{k} -h"
-    end
-
-    # We don't test ssl-setup here, because its actions without any arguments
-    # are tested elsewhere.
-    next if k == 'ssl-setup'
-    step "running puppetdb-#{k} with no arguments should not error out" do
-      on database, "#{bin_loc}/puppetdb #{k}", :acceptable_exit_codes => [1]
-      assert_match(/Missing required argument/, stdout)
-    end
+  step "running 'puppetdb ssl-setup -h' should not error out" do
+    on database, "#{bin_loc}/puppetdb ssl-setup -h"
```

Comment:
Perhaps Ruby expansion handles this somehow, or it's not something we want to pursue at the moment, but I'm wondering what happens if bin_loc has spaces in it, or something even unfriendlier.